spec/inquisition/rails_best_practices/categorizer_spec.rb

```diff
@@ -0,0 +1,54 @@
+RSpec.describe Inquisition::RailsBestPractices::Categorizer do
+  describe '#find_category' do
```

Comment:
`describe '.find_category' do` instead of `describe '#find_category' do`
https://github.com/rubocop-hq/rspec-style-guide#example-group-naming